### PR TITLE
Kg 262 publish datamodels on developer meemoo be

### PR DESCRIPTION
**Pull request by mielvds**

### CODE REVIEW

File: description/audiovisual.shacl.ttl [hunk 1]
```diff
@@ -32,17 +31,17 @@
...
     ] ;
-    dct:title     "Data model Description"@en ;
-    dct:title     "Modèle de données Description"@fr ;
-    dct:title     "Datamodel Descriptie"@nl ;
-    dct:created   "2022-01-07" ;
+    dct:title     "Data model Audiovisual archive objects"@en ;
+    dct:title     "Modèle de données Objets d'archive audiovisuelle"@fr ;
+    dct:title     "Datamodel Audiovisuele archiefobjecten"@nl ;
+    dct:created   "2024-01-09" ;
     dct:modified  "2024-01-17" ;
```
Comment (milvld) [inline description/audiovisual.shacl.ttl:38]:
```suggestion
    dct:modified  "2024-02-13" ;
```

File: description/audiovisual.shacl.ttl [hunk 2]
```diff
@@ -1,23 +1,22 @@
...
-
-
 @prefix rel: <http://id.loc.gov/vocabulary/preservation/relationshipSubType/> .
 @prefix premis: <http://www.loc.gov/premis/rdf/v3/> .
 
-<http://data.hetarchief.be/ns/description/>  a owl:Ontology ;
+@base <https://data.hetarchief.be/ns/audiovisual> . 
+
+<>  a owl:Ontology ;
     vann:preferredNamespacePrefix "haDes" ;
```
Comment (milvld) [inline description/audiovisual.shacl.ttl:20]: Should we also change this to a separate prefix then? The specific ontology terms are part of the general ontology 'haDes', but the data model is separate. I don't know what makes the most sense anymore :D

File: description/bibliographic.shacl.ttl [hunk 3]
```diff
@@ -28,17 +30,17 @@
...
     ] ;
-    dct:title     "Data model Description"@en ;
-    dct:title     "Modèle de données Description"@fr ;
-    dct:title     "Datamodel Descriptie"@nl ;
-    dct:created   "2022-01-07" ;
+    dct:title     "Bibliographic data model"@en ;
+    dct:title     "Modèle de données bibliographiques"@fr ;
+    dct:title     "Bibliografisch datamodel "@nl ;
+    dct:created   "2023-11-13" ;
     dct:modified  "2024-01-17" ;
```
Comment (milvld) [inline description/bibliographic.shacl.ttl:37]:
```suggestion
    dct:modified  "2024-02-13" ;
```

File: description/description.rdfs.ttl [hunk 4]
```diff
@@ -31,7 +31,7 @@ haDes:NewspaperIssuePage   a   rdfs:Class ;
     rdfs:isDefinedBy <https://data.hetarchief.be/ns/description/> .
 
 haDes:DVD   a   rdfs:Class ;
-    rdfs:label "Dvd"@nl ;
+    rdfs:label "DVD"@nl ;
```
Comment (milvld) [inline description/description.rdfs.ttl:34]: In het NL is het normaliter met kleine letters: https://woordenlijst.org/zoeken/?q=dvd

File: description/description.rdfs.ttl [hunk 5]
```diff
@@ -40,8 +40,8 @@ haDes:DVD   a   rdfs:Class ;
     rdfs:isDefinedBy <https://data.hetarchief.be/ns/description/> .
 
 haDes:DVDChapter   a   rdfs:Class ;
-    rdfs:label "Dvd-hoofdstuk"@nl ;
-    rdfs:label "DVD Chapter"@en ;
+    rdfs:label "DVD-hoofdstuk"@nl ;
```
Comment (milvld) [inline description/description.rdfs.ttl:43]: In het NL is het normaliter met kleine letters: https://woordenlijst.org/zoeken/?q=dvd
```suggestion
    rdfs:label "Dvd-hoofdstuk"@nl ;
```

File: description/description.rdfs.ttl (hunk 4 above)
Comment (milvld) [inline description/description.rdfs.ttl:34]: In het NL is het normaliter met kleine letters: https://woordenlijst.org/zoeken/?q=dvd

```suggestion
    rdfs:label "Dvd"@nl ;
```

File: description/film.shacl.ttl [hunk 6]
```diff
@@ -32,14 +32,16 @@
...
         schema:email  "milan.valadou@meemoo.be"
     ] ;
-    dct:title     "Data model Description"@en ;
-    dct:title     "Modèle de données Description"@fr ;
-    dct:title     "Datamodel Descriptie"@nl ;
+    dct:title     "Data model Film"@en ;
+    dct:title     "Modèle de données Film"@fr ;
+    dct:title     "Datamodel Film"@nl ;
+    dct:created   "2023-11-09" ;
+    dct:modified  "2024-01-08" ;
```
Comment (milvld) [inline description/film.shacl.ttl:39]:
```suggestion
    dct:modified  "2024-02-13" ;
```

**Review by milvld — APPROVED** [review]

File: description/audiovisual.shacl.ttl (hunk 2 above)
Comment (mielvds) [reply description/audiovisual.shacl.ttl:20]: see comment in https://meemoo.atlassian.net/browse/KG-262?focusedCommentId=46431 Not sure either 😄

Comment (mielvds) [reply description/audiovisual.shacl.ttl]: I went for a compromise and removed them. They were currently not serving any purpose and they can be brought back if they do.

**Pull request merged**